frontend/class-frontend.php

```diff
@@ -1879,6 +1881,17 @@ private function is_premium() {
 	}
 
 	/**
+	 * Checks whether the user has written a meta-description. If written,  makes sure meta robots content is noodp.
+	 *
+	 * @param String $description The content of the meta description
+	 */
+	private function add_robot_content_noods( $description ) {
+		if ( ! ( $description === '' ) && $this->options['noodp'] === false ) {
```

Comment:
Why not check if `$description` is [empty](http://php.net/manual/en/function.empty.php) instead of `''`?